app/main.py

```diff
@@ -1,214 +1,25 @@
-KNIGHTS = {
```

Comment:
where did the config go? I would put it in a separate file and then import from there